src/com/pentalog/pentastagiu/homework2/Shop/Basket.java

```diff
@@ -0,0 +1,38 @@
+package com.pentalog.pentastagiu.homework2.Shop;
+
+public class Basket {
+    private Customer customer;
+    private double totalPrice;
+
+    public Basket(Customer customer) {
+        this.customer = customer;
+    }
+
+    public double getTotalPrice() {
+
+        double discount = 0;
+        switch(customer.getMembership()) {
+            case NOMEMBERSHIP:
+                discount = 0.0;
+                break;
+            case SILVER:
+                discount = 0.1;
+                break;
+            case GOLD:
+                discount = 0.2;
```

Comment:
pt ca valoarea discountului este strans legata de tipurile de membership, ar putea fi mutata la nivelului enum-ului si astfel ai avea un singur loc in care controlezi relatia asta